runtime/doc/lsp.txt

```diff
@@ -699,6 +699,24 @@ buf_detach_client({bufnr}, {client_id})          *vim.lsp.buf_detach_client()*
       • {bufnr}      (integer) Buffer handle, or 0 for current
       • {client_id}  (integer) Client id
 
+buf_inlay_hint_disable({bufnr})             *vim.lsp.buf_inlay_hint_disable()*
+    Disable inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}  (integer|nil) Buffer handle, or nil for current
+
+buf_inlay_hint_enable({bufnr})               *vim.lsp.buf_inlay_hint_enable()*
```

Comment:
- "enable" can take an `enable` parameter, then you don't need the `_disable` variant.
- won't be long before someone asks for a getter (`is_enabled`), right? So if we add that now, then we also don't need `toggle()`.

Reply:
What if we don't add a getter and make it so that it toggles by default if there is no `enable` parameter, and enables/disables if there is?

Reply:
Anything that results in 1 function instead of 3 is good enough.

Reply:
done